__tests__/app.test.js

```diff
@@ -78,4 +78,32 @@ describe("/api/articles/:article_id", () => {
         expect(res.body.msg).toBe('Oh no! Please enter a valid article ID!');
       });
   });
+})
+
+describe("/api/articles", () => {
+  test("GET - status 200 - Returns all articles", () => {
+    return request(app)
+    .get("/api/articles")
+    .expect(200)
+    .then((res) => { res.body.articles.forEach( (articles) => {
+      expect(articles).toEqual(
```

Comment:
Please check the length of the 'articles' array first to avoid false positive tests.

Reply:
No problem, I'll add that now!